actionview/lib/action_view/test_case.rb

```diff
@@ -42,6 +42,160 @@ def initialize
       end
     end
 
+    module Decoding
+      extend ActiveSupport::Concern
+
+      class_methods do
+        def inherited(descendant)
+          super
+
+          descendant.class_attribute :decoders, default: decoders.dup
+        end
+
+        # Register a callable to decode rendered content for a given MIME type
+        #
+        # Each registered decoder will also define a <tt>#rendered_$MIME</tt> helper
+        # method, where <tt>$MIME</tt> corresponds to the value of the
+        # <tt>mime</tt> argument
```

Comment:
It seems the `$MIME` gets removed by RDoc:
```bash
 echo "define a <tt>#rendered_$MIME</tt> helper" | rdoc --pipe
<p>define a <code>#rendered_</code> helper</p>
```

In https://github.com/rails/rails/pull/48193 I used the form `#rendered_*` for dynamic parts, but `#rendered_[mime]` has also been used.

We can also use short `+` notation here:

```suggestion
        # Each registered decoder will also define a +#rendered_[MIME]+ helper
        # method, where +[MIME]+ corresponds to the value of the
        # +mime+ argument
```